### DISCUSSION TITLE
Questions on Flowise features and documentation

### DISCUSSION THREAD

**Discussion by chrbrnracn** [FlowiseAI/Flowise#2423] [Q&A]
Hi,

I have some questions regarding features of Flowise. I checked the documentation but couldn't find entries for some of the nodes and configuration options.

1. When creating a chatflow I find it difficult to find the right nodes and how to configure them. For example
   - What's the difference between Chat Models/Embeddings als LLMs. I found Ollama in all of them but LocalAI only in Chat Models and Embeddings.
   - Where can I find the documentation for the PostgreSQL vector store? (took me half a day to figure out how to accept self-signed SSL certificates)

2. On the landing page there's the "Document Stores" item. I couldn't find that in the documentation. From clicking around it looks like I can configure a "one-shot" data ingestion.
   - Where is this the storage location of this document store (so I can mount a volume when I run Flowise in a container)
   - What kind of store is this?
   - Can I also configure scheduled scrapes, e.g. for a web page or Confluence where I know content is regularly updated?
   - Which nodes can I connect to the "Document Store" node output?

3. Do Buffer Memory nodes work only inside that specific workflow or can I use them across workflows, e.g. by specifying the same session id?

3. In general some kind of overview like a table with filter function would be helpful where one can look up which nodes can be connected as input/output to which other nodes.

---

And a more general question... is Flowise intended to be used behind a separate frontend only? To be more specific: one of my use cases is to create an OpenAI compatible API frontend in an air-gapped environment for access from IDE plugins.
- Is there a recommendation how to secure the endpoint with user specific API keys?
- Is there a node or template which provides an OpenAI compatible endpoint or do I have to build that on my own?

Kind Regards
Christian

**Comment by toi500** [comment]
Regarding the documentation, it's significantly behind production, with nearly a hundred nodes missing. I contacted the team last week, suggesting a temporary solution of including these nodes in the docs for basic awareness of their availability.

I have documented (name, image and description) all the currect nodes here: https://toi500.gitbook.io/flowise-docs/integrations (I'm still waiting for a response to my pull request).

Right now, the only way to get technical details of the new nodes is by searching on https://js.langchain.com/v0.2/docs/introduction/

**Comment by crashr** [comment]
@chrbrnracn 

> one of my use cases is to create an OpenAI compatible API frontend

To a certain extend that can be covered with this https://github.com/crashr/open-adapter
It's still work in progress and far from production ready but it works for me.
